java/client/test/com/thoughtworks/selenium/BaseSuite.java

```diff
@@ -35,13 +35,13 @@
   public static ExternalResource testEnvironment = new ExternalResource() {
     @Override
     protected void before() {
-      log.info("Preparing test environment");
+      log.finest("Preparing test environment");
```

Comment:
This is in test code: understanding what we're doing is important in this context.